packages/core/src/Cardano/types/Transaction.ts

```diff
@@ -82,9 +81,29 @@ export interface Redeemer {
 
 export type Signatures = Map<Ed25519PublicKey, Ed25519Signature>;
 
-export type Witness = Omit<Partial<Ogmios.TxAlonzo['witness']>, 'redeemers' | 'signatures'> & {
+export declare type Signature = string;
+export declare type ChainCode = string;
+export declare type AddressAttributes = string;
+export declare type VerificationKey = string;
```

Comment:
Not 100% sure about this one
```suggestion
export declare type VerificationKey = Ed25519PublicKey;
```

Reply:
I made the change here: https://github.com/input-output-hk/cardano-js-sdk/pull/388/commits/f93bd1a2bac1896cc1289c422db283dd6dfd92dd